resources/js/Pages/Admin/Rooms/CreateRoomForm.vue

```diff
@@ -45,12 +76,75 @@
                         <jet-input-error :message="createRoomForm.error('status')" class="mt-2"/>
                     </div>
 
-                    <div class="col-span-6 sm:col-span-6">
+                    <div class="col-span-6 sm:col-span-3">  
+                        <jet-label for="equippment" value="Equippment"/>
+                        <br>
+                        <label class="flex items-center">
+                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.projector"/>
+                        <span class="ml-2 text-sm text-black">Projector</span>    
+                        </label> 
+                        <label class="flex items-center">
+                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.television"/>
+                        <span class="ml-2 text-sm text-black">Television</span>  
+                        </label> 
+                        <label class="flex items-center">
+                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.computer"/>
+                        <span class="ml-2 text-sm text-black">Computer</span>
+                        </label>   
+                        <label class="flex items-center"> 
+                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.whiteboard"/>
+                        <span class="ml-2 text-sm text-black">Whiteboard</span>    
+                        </label>                 
+                        <jet-input-error :message="createRoomForm.error('equippment')" class="mt-2"/>
+                    </div>
+
+                    <div class="col-span-6 sm:col-span-3">  
+                        <jet-label for="restrictions" value="Restrictions"/>
+                        <br>
+                        <label class="flex items-center">
+                        <input type="checkbox" class="form-checkbox" v-model="createRoomForm.ambiant_music"/>
+                        <span class="ml-2 text-sm text-black">Ambiance Music</span>    
```

Comment:
What does a Ambiance Music Restriction refer to?

Reply:
Some spaces allow for ambiance music to be played others are considered more 'quiet' spaces (from "summary of csu administered spaces" in slack/comms-stakeholder tab).